_raml/apps/products/schema/patch_product_productid_customattributes_request_schema.json

```diff
@@ -0,0 +1,11 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Request Schema",
```

Comment:
See comment above :)

Reply:
@AliceStatler Let's see what @mhoehn has to say to the chase above.